Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py

```diff
@@ -128,18 +128,9 @@ def detect_type(indicator):
     if re.match(ipv6Regex, indicator):
         return FeedIndicatorType.IPv6
 
-    if re.match(sha256Regex, indicator):
-        return FeedIndicatorType.SHA256
```

Comment:
should be big if with all the options and then sent as FeedIndicatorType.File

Reply:
done